lib/turn.rb

```diff
@@ -0,0 +1,23 @@
+class Turn
+
+  attr_reader :guess, 
+              :card, 
+              :validity
+
+  def initialize(guess, current_card)
+    @guess = guess
+    @card = current_card
+  end
+
+  def correct?
+    @validity = @card.answer.include?(@guess)
```

Comment:
I would consider updating this to `downcase` the `answer` and the `@guess` to try and allow a little more flexibility for the user.